content/blog/2020-05-08-dvc-ambassador-program.md.md

```diff
@@ -0,0 +1,151 @@
+---
+title: 'Join the DVC Ambassador Program!'
+date: 2020-05-08
+description: |
+  We're launching our ambassador program for people all around the world to get involved in 
+  the DVC community.
+descriptionLong: |
+  We're launching our ambassador program for people all around the world to get involved in 
```

Comment:
I think we should add some summary here, some brief agenda, or something?

Reply:
`We're launching our ambassador program for people all around the world to get involved in 
  the DVC community. Our first ambassador, Marcel Ribeiro Dantas, shares a guest blog about 
  how ambassadors support open source projects through blog writing, public outreach, and code.`

Reply:
Marcel Ribeiro-Dantas. Don't forget the dash please hahaha